package.json

```diff
@@ -2,14 +2,30 @@
 	"name": "@cloudflare/next-on-pages",
 	"version": "1.0.0",
 	"bin": "./bin/index.js",
+	"exports": {
+		"./utils": "./dist/utils/index.js"
+	},
+	"typesVersions": {
+		"*": {
+			"utils": [
+				"./dist/utils/index.d.ts"
+			]
+		}
+	},
 	"scripts": {
 		"prettier": "npx prettier --ignore-unknown --ignore-path=.gitignore .",
 		"prettier:check": "npm run prettier -- --check",
 		"prettier:fix": "npm run prettier -- --write",
 		"lint": "eslint src",
 		"types-check": "tsc --noEmit",
-		"build": "npx esbuild --bundle --platform=node ./src/index.ts --external:esbuild --external:chokidar --outfile=./dist/index.js",
-		"build:watch": "npm run build -- --watch",
+		"build-cli": "npx esbuild --bundle --platform=node ./src/cli/index.ts --external:esbuild --external:chokidar --outfile=./dist/index.js",
+		"build-utils": "npx esbuild --bundle --platform=node --format=esm ./src/utils/index.ts --external:esbuild --external:chokidar --outfile=./dist/utils/index.js",
+		"build-utils-types": "tsc  -p ./tsconfig.utils.json --emitDeclarationOnly --declaration --outDir ./dist/utils",
+		"build": "npm run build-cli ; npm run build-utils ; npm run build-utils-types",
```

Comment:
Just a suggestion :) https://www.npmjs.com/package/concurrently

Reply:
yeah, I didn't want to introduce an extra dependency but this could be made a bit nicer

Since you're suggesting it I'll try something :slightly_smiling_face: 

I was actually thinking of using [npm-run-all](https://www.npmjs.com/package/npm-run-all) and do:
```json
  "build": "run-p build-*",
```

Reply:
I've added `npm-run-all` and made the `build` and `build:watch` commands simpler, please have a look 🙂

(note: I used a somewhat weird naming convention, the fact is that in the `run-p` pattern I needed to have distinct prefixes for the build and build+watch scripts)

Reply:
npm-run-all is no longer maintained (whereas concurrently has basically all the same functionality and is maintained :))

Reply:
ok, concurrently it is then 🙂👍 

By the way, I wanted `npm-run-all` because you could use `*` I didn't think that concurrently supported that but it looks like it does, so yeah I'm totally ok with that 🙂 (actually their matching seems even more advanced 😄)

Reply:
switched to `concurrently` it works great 😃